index.html

```diff
@@ -42,187 +42,231 @@ <h4>I’am a Software Developer</h4>
+                website Look through some of my work and experience! If you like what you see and have a project you
+                need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="about-btn"><span class="about-btn-title">Get My Resume</span></a>
         </div>
-        <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-        </ul>
-        <div class="skill-frame-1 left-skill">
-            <img class="img-rectangle" src="/images/square.png" alt=""> <span class="img-title">Frameworks</span>
+        <div class="divider">
+            <div class="vector-1"></div>
         </div>
-        <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>Rspec</li>
-            <li>CampyBara</li>
-            <li>Selenium</li>
-        </ul>
-        <div class="skill-frame-1 right-skill">
-            <img class="img-rectangle" src="/images/circle.png" alt=""> <span class="img-title">Skills</span>
+        <!-- Skill div Start -->
+        <div class="skill">
+            <div class="skill-frame-1">
+                <img class="img-rectangle" src="/images/rectangle.png" alt=""> <span class="img-title">Language</span>
+            </div>
+            <ul>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+            <div class="skill-frame-1 left-skill">
+                <img class="img-rectangle" src="/images/square.png" alt=""> <span class="img-title">Frameworks</span>
+            </div>
+            <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>Rspec</li>
+                <li>CampyBara</li>
+                <li>Selenium</li>
+            </ul>
+            <div class="skill-frame-1 right-skill">
+                <img class="img-rectangle" src="/images/circle.png" alt=""> <span class="img-title">Skills</span>
+            </div>
+            <ul>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
         </div>
-        <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-        </ul>
-    </div>
-    <!-- Skill div End -->
-</section>
-<!-- About Section End -->
+        <!-- Skill div End -->
+    </section>
+    <!-- About Section End -->
+
+    <!-- Contact Form Start -->
+
+
+    <section>
+
+        <div id="maincontact">
+            <div class="contact">
+                <h2>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in
+                    touch.</h2>
+            </div>
+            <div class="form-contact">
```

Comment:
- [ ] Since `<form>` is block level element, do we need a `<div>` wrapper here? I believe you do the styling without it.